make/kz.mk

```diff
@@ -74,7 +74,7 @@ test/$(PROJECT).app: ERLC_OPTS += -DTEST
 test/$(PROJECT).app: $(TEST_SOURCES)
 	@mkdir -p test/
 	@mkdir -p ebin/
-	ERL_LIBS=$(ELIBS) erlc -v +nowarn_missing_spec $(ERLC_OPTS) $(TEST_PA) -o ebin/ $(TEST_SOURCES)
+	ERL_LIBS=$(ELIBS) erlc -v +nowarn_missing_spec +nowarn_export_all $(ERLC_OPTS) $(TEST_PA) -o ebin/ $(TEST_SOURCES)
```

Comment:
We prefer to export specific functions rather than using export_all as we believe its usage leads to bad practice :)